Java/other/SwapNibbles.java

```diff
@@ -0,0 +1,31 @@
+import java.util.Scanner;
+
+public class SwapNibbles {
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter a number:");
+        int num = sc.nextInt();
+
+        int result = swapNibbles(num);
+
+        System.out.println("Number after swapping nibbles:" + result);
+        sc.close();
+    }
+
+    private static int swapNibbles(int x) {
+        // (x & 0x0F) << 4 gives us the last 4 bits of the number and
```

Comment:
Can we please use multi-line comment i.e `/* */` here from (line 17-20)?

Reply:
@Kajol-Kumari done. Please have a look and merge if looks good.